org.civicrm.reqangular/src/common/angular-date/src/services/DateFactory.js

```diff
@@ -0,0 +1,20 @@
+var moment = require('../../../vendor/moment.min.js');
+
+function DateFactory(){
```

Comment:
I'm not sure I understand this. If `DateFactory` is just a wrapper around moment.js that does nothing else other than passing the given params to the library (but maybe i'm reading the code wrong), why not use moment.js directly then?